app/controllers/spree/admin/base_controller_decorator.rb

```diff
@@ -53,7 +53,7 @@ def unauthorized
   def active_distributors_not_ready_for_checkout
     ocs = OrderCycle.managed_by(spree_current_user).active
     distributors = ocs.map(&:distributors).flatten.uniq
-    Enterprise.where('id IN (?)', distributors).not_ready_for_checkout
+    Enterprise.where('enterprises.id IN (?)', distributors).not_ready_for_checkout
```

Comment:
Why do we need the table? AR infers it from the class you are calling to.

Reply:
because of the scope not_ready_for_checkout the query errors out with "ambiguous field id". this change fixes that problem in 2-1-stable.